CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Prometheus charm.
+
+- It is recommended that prior to developing enhancements to this charm
+  an issue explaining the use case is created at the official
+  Canonical [github repository](https://github.com/canonical/prometheus-operator).
+- Should you require any assistance in exploring use cases or
+  discussing proposed implementation you may directly engage with
+  other developers in the Canonical
+  [public channel](https://chat.charmhub.io/charmhub/channels/charm-dev) using
+  [Mattermost](https://mattermost.com/).
+- It is strongly recommended that prior to engaging in any enhancements
+  to this charm you familiarise your self with the Juju
+  [SDK](https://juju.is/docs/sdk) and
```

Comment:
"- Familiarising yourself with the [Juju Operator SDK](https://juju.is/docs/sdk) and the Python
  [Operator Framework](https://ops.readthedocs.io/en/latest/) library will help you a lot when working on PRs.

Reply:
Fixed.